apps/discord-bot/src/commands/leaderboards/leaderboard.profile.tsx

```diff
@@ -32,6 +33,7 @@ export interface LeaderboardProfileProps {
   data: LeaderboardData[];
   t: LocalizeFunction;
   type: LeaderboardType;
+  time?: HistoricalType | undefined;
```

Comment:
```suggestion
  time?: HistoricalType;
```
I would use the new enum here too